.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Shouldn't this file be ignored?

Reply:
Good question.
Added it to allow consistency among devs which use vscode and want to debug in the same circumstances.

Don't know if should add or not. What do you think?

Reply:
It's fine by me.

Reply:
Leave it, then.